src/components/Icon/Icon.css

```diff
@@ -0,0 +1,42 @@
+@import "../../styles/theme/index.css";
+
+
+:root {
+  --icon-border-active: 1px solid var(--primary-color-900);
+  --icon-border-hover: none;
+
+  --icon-filter-color: none;
+  --icon-filter-color-focus: none;
+  --icon-filter-color-hover: none;
+}
+
+.offscreen-button {
```

Comment:
I'm curious, why are these classes called `offscreen` ?

Reply:
Oh, offscreen should be just the description, I'll fix it